.pylintrc

```diff
@@ -1,5 +1,5 @@
 [MESSAGE CONTROL]
-disable = abstract-method,missing-docstring,redefined-outer-name,too-few-public-methods
+disable = abstract-method,inconsistent-return-statements,missing-docstring,redefined-outer-name,too-few-public-methods
```

Comment:
Pylint ended up upgraded, could downgrade or fix the problem.

Reply:
Can't downgrade, version enforce by Proofreader.

Reply:
Whats causing the `inconsistent-return-statements` message?

Reply:
Can't remember which files, it's where not all functions have return statements.